.github/workflows/e2e-test-tune-api.yaml

```diff
@@ -0,0 +1,41 @@
+name: E2E Test with tune API
+
+on:
+  pull_request:
+    paths-ignore:
+      - "pkg/ui/v1beta1/frontend/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Test Env
+        uses: ./.github/workflows/template-setup-e2e-test
+        with:
+          kubernetes-version: ${{ matrix.kubernetes-version }}
+      
+      - name: Setup Minikube Cluster
+        shell: bash
+        run: ./test/e2e/v1beta1/scripts/gh-actions/setup-minikube.sh
+
+      - name: Setup Katib
+        shell: bash
+        run: ./test/e2e/v1beta1/scripts/gh-actions/setup-katib.sh
+
+      - name: Run E2E Experiment
+        shell: bash
+        run: ./test/e2e/v1beta1/scripts/gh-actions/run-e2e-tune-api.sh
```

Comment:
Could you reuse the existing workflows similar to https://github.com/kubeflow/katib/blob/51b246fa1c23db130c373fa9749897a0d0b334ab/.github/workflows/e2e-test-darts-cifar10.yaml#L17-L31?

I think that we can do it once we add additional parameters like `testing-scripts` here: https://github.com/kubeflow/katib/blob/51b246fa1c23db130c373fa9749897a0d0b334ab/.github/workflows/template-e2e-test/action.yaml#L5

Reply:
Okay, that sounds good.